docs/events.md

```diff
@@ -68,6 +69,12 @@ Resource      |Event    |Event Type
 `PipelineRun` | `Condition Change while Running` | `dev.tekton.event.pipelinerun.unknown.v1`
 `PipelineRun` | `Succeed` | `dev.tekton.event.pipelinerun.successful.v1`
 `PipelineRun` | `Failed`  | `dev.tekton.event.pipelinerun.failed.v1`
+`Run`         | `Started` | `dev.tekton.event.run.started.v1`
```

Comment:
what's the reason not to send a condition change while running event? is this because we don't know what the run's previous status was, and can't make assumptions based on what it currently is? Is this a feature users might want? I'm a bit confused because the PR description describes an event cache but I'm not really seeing where that happens.

Reply:
We cannot make assumptions about the `Run` and it will use its `Conditions` beyond what we specify in the custom task interface. So we can see that the resource has been created (start), we can see that an initial condition was set (running) and we can see when it's finished (successfully or not).